docs/interfaces.html

```diff
@@ -0,0 +1,272 @@
+        <li>Similarly, a wallet implementation which relies on an external storage, such as encrypted data vaults, might choose not to implement <code>Export</code>, <code>Import</code>, <code>Lock</code>, and/or <code>Unlock</code> interfaces.</li>
+    </ul>
+
+    </p>
+
+    <section>
+        <h3>Import</h3>
+        <p>
+        Takes a serialized exported wallet representation as input. Loads the
+        representation into wallet software.
+        </p>
+        <p>
+        This method may not be necessary for implementations that rely on
+        external storage, such as Encrypted Data Vaults.
+        </p>
+    </section>
+
+    <section>
+        <h3>Export</h3>
+
+        <p>Only ciphertext wallet contents can be exported.</p>
+
+        <p>Produces a serialized exported wallet representation.</p>
+
+        <p>
+            This method may not be necessary for implementations that rely on
+            external storage, such as Encrypted Data Vaults.
+        </p>
+    </section>
+
+
+    <section>
+        <h3>Unlock</h3>
+        <p>
+        Transforms one or more wallet contents from ciphertext to plaintext.
+        </p>
+
+        <p>Requires knowledge of the password used to lock the wallet.</p>
+
+        <p>
+        This method may not be necessary for implementations that rely on
+        external storage, such as Encrypted Data Vaults.
+        </p>
+    </section>
+
+    <section>
+        <h3>Lock</h3>
+        <p>
+        Transforms one or more wallet contents from plaintext to ciphertext.
+        </p>
+        <p>Requires knowledge of the password used to lock the wallet.</p>
+
+        <p>
+        This method may not be necessary for implementations that rely on
+        external storage, such as Encrypted Data Vaults.
+        </p>
+
+        <div class="issue" data-number="5">
+        Mostly a reference implementation issue. Locked representation should
+        indicate a best practice.
+        </div>
+    </section>
+
+    <section>
+        <h3>SignRaw</h3>
+        <p>
+        Takes a Buffer as input, and an <code>options</code> object, which
+        contains at least a <code>verificationMethod</code>, and may content
+        additonal details such as <code>alg</code> or other JOSE related
+        terms.
+        </p>
+
+        <p>Must support detached signatures.</p>
+
+        <p>
+            This method may not be necessary for implementations that rely on
+            other dedicated interface function to perform signing.
+
+            For example, a verifiable credential wallet that adds proofs using <code>Issue</code>, <code>Prove</code> interfaces might not implement this interface.
+        </p>
+    </section>
+
+    <section>
+        <h3>VerifyRaw</h3>
+        <p>
+        Takes a Buffer as input, and an <code>options</code> object, which
+        contains at least a <code>verificationMethod</code>, and may content
```

Comment:
more sure. confirms the above is correct.
```suggestion
        contains at least a <code>verificationMethod</code>, and may contain
```